app/assets/dashboard.css

```diff
@@ -0,0 +1,123 @@
+table.dashboardpanel {
+    border: 2px solid black;
+    border-collapse: collapse;
+    font: 0.8em "Lucida Console",Monaco,monospace;
+    border-spacing: 3px;
+    overflow: scroll;
+//  width:450px;
+}
+p.footnote {
+    font-size:0.6em;
+    font-style:italic;
+}
+td {
+    border: 1px solid black;
+    padding: 3px;
+}
+td.link {
+    color: #0000ff;
+  text-decoration:none;
+    cursor: pointer;
+}
+td.link:hover {
+    text-decoration:underline;
+}
+
+tr.dashboardheader {
+    font-size:1.1em;
+    font-weight: bold;
+    background-color: #AAAAFF;
+}
+
+div.dashboardpanel {
+    /*width:200px; */
+    height:100%;
+    padding:10px;
+    float: left;
+    overflow: auto;
+}
+
+div.toprow {
+    display:table;
+    padding:5px;
+    height: auto;
+    width: 100%;
+}
+
+div.bodyrow {
+    display:block;
+    width:100%;
+    height:auto;
+}
+
+.selector_menu {
+    border: none;
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    border-bottom: 1px solid black;
+    border-collapse: collapse;
+    cursor: pointer;
+}
+
+td.selector_menu:hover {
+    color: #0000ff;
+    border-left: 3px solid #0000aa;
+    text-decoration:underline;
+}
+
+.selectable:hover {
+   background-color: #CCCCCC;
+   cursor:pointer;
+}
+
+.status_success {
```

Comment:
`status--success` would be better [BEM](http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/)